web/lib/hook/apolloClient.tsx

```diff
@@ -1,67 +1,14 @@
 // from https://github.com/awslabs/aws-mobile-appsync-sdk-js#creating-a-client
 
-import { ApolloClient, ApolloLink, ApolloProvider, createHttpLink, InMemoryCache } from '@apollo/client';
-import { SentryLink } from 'apollo-link-sentry';
-import { AUTH_TYPE, createAuthLink } from 'aws-appsync-auth-link';
-import { createSubscriptionHandshakeLink } from 'aws-appsync-subscription-link';
-import { useSession } from 'next-auth/react';
+import { ApolloProvider } from '@apollo/client';
 import React from 'react';
-import { APPSYNC_ENDPOINT, REGION } from '../config/next';
+import { getApolloClient } from '../client/apollo';
 
 export function useApolloClient() {
-  const session = useSession();
-  session.data?.user;
-
-  const getCognitoAccessJwt = React.useCallback(async () => session.data?.user?.accessToken || '', [session]);
-
-  return getApolloClient(getCognitoAccessJwt);
+  return React.useMemo(() => getApolloClient(), []);
```

Comment:
Is there any reason to have this hook? Is useMemo redundant here?

Reply:
so when we pass it to down it will not trigger some unwanted re-renders.
Removing useMemo would break things, but may influence performance.

Reply:
also when you are passing something to provider it should be memoized or you will trigger some "random" re-renders.